### ISSUE TITLE
Add a skip-nav link

### ISSUE THREAD

**Issue by 1Copenut** [1Copenut/c3-eleventy#33]
Apparently in the rush to block out the elements, I forgot to build a skip-nav link in. Since I'm not supporting IE11, this should be fairly straightforward.
